src/NDItk/base/StringListRecord/src/write.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ *  @brief Write the record content
+ *
+ *  This assumes that the content is defined.
+ *
+ *  @param[in] iter   the current position in the output
+ */
+template< typename OutputIterator >
+void write( OutputIterator& iter ) const {
+
+  std::ostringstream buffer;
+
+  auto full = this->data().size() / 5;
+  auto partial = this->data().size() - full * 5;
+  auto x = this->data().begin();
+
+  while ( full-- ) {
+
+    buffer << "    " << x[0];
+    buffer << ' ' << std::setprecision( 15 ) << x[1];
```

Comment:
Might be put in later PRs: why is there precision for strings?

Reply:
Copy/paste. I'm an idiot who works too fast.